.github/workflows/ci-pr-validation.yml

```
@@ -0,0 +1,72 @@
+name: CI - PR Validation
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  coderabbit-gate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Coderabbit status
+        uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)